x-pack/plugins/cases/public/components/custom_fields/form.test.tsx

```diff
@@ -164,7 +170,10 @@ describe('CustomFieldsForm ', () => {
       'value',
       customFieldsConfigurationMock[1].label
     );
-    expect(await screen.findByTestId('text-custom-field-options')).not.toHaveAttribute('checked');
+    expect(await screen.findByTestId('toggle-custom-field-required')).toHaveAttribute('checked');
```

Comment:
The test says `deserializes the data correctly if required not selected` but the test selects the required field. This seems more than a fix of a test because the `customFieldsConfigurationMock` changed rather than intentionally testing the required field.

Reply:
The names are wrong, I should have updated them when I added the extra `expect`s here.

In this test and the one above, I wanted to make sure that all values for `text`(previous test) and `toggle`(this test) are deserialized correctly.